xgcm/test/test_metric_ops_multi_axis.py

```diff
@@ -0,0 +1,218 @@
+from __future__ import print_function
+import pytest
+
+from xgcm.grid import Grid
+from xgcm.test.datasets import datasets_grid_metric
+
+
+def _expected_result(da, metric, grid, dim, axes, funcname):
```

Comment:
After this is merged I could consolidate the derivative tests to this module if you agree.

Reply:
I did that (I think) in `test_metric_ops_single_axis.py`. The idea here is to seperate the operations that can be executed on one axis after another (e.g. derivative, interp) and others that need to consider multidimensional metrics. This will probably have to be generalized if we add the finite volume ones (I have a plan how to do this elegantly, but will postpone until after the hackathon).

Reply:
I guess I only renamed the `test_derivative` for now. This could def need some work afterwards!

Reply:
Ah I see now. Yeah going to FV will require some generalization but it should be do-able. Thanks